prime-router/src/test/kotlin/CsvFileTests.kt

```diff
@@ -51,16 +60,21 @@ class CsvFileTests {
     private fun transformFileAndTest(fileName: String) {
         val file = File(fileName)
         val baseName = file.name
-        assertTrue(file.exists())
+        assertThat(file).exists()
         val schema = metadata.findSchema(defaultSchema) ?: error("$defaultSchema not found.")
 
         // 1) Ingest the file
         val result = csvSerializer.readExternal(schema.name, file.inputStream(), TestSource)
         assertTrue(result.warnings.isEmpty() && result.errors.isEmpty())
+        assertThat(result).all {
+            prop("warnings") { ReadResult::warnings.call(it) }.isEmpty()
+            prop("errors") { ReadResult::errors.call(it) }.isEmpty()
+        }
+        assertThat(result).hasNoWarnings().hasNoErrors()
```

Comment:
This shows two different ways to accomplish the same thing. One is by using `assertk`'s prop method, which lets you reflect into the object and assert against it. The second one uses two extension methods.